mongo/oplog.go

```diff
@@ -200,6 +199,9 @@ func (t *OplogTailer) Dying() <-chan struct{} {
 // Stop shuts down the OplogTailer. It will block until shutdown is
 // complete.
 func (t *OplogTailer) Stop() error {
+	if err := t.tomb.Err(); err != tomb.ErrStillAlive {
```

Comment:
Why do we need this check?
I can't see offhand what it's for

Reply:
Ah, no we don't. Stems from a misunderstanding that I had earlier.